api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentGroupController.java

```diff
@@ -209,8 +212,8 @@ public PaymentGroupController(PaymentGroupService paymentGroupService, PaymentGr
         }
 
         // trigger Apportion based on the launch darkly feature flag
-        boolean apportionFeature = featureToggler.getBooleanValue("apportion-feature",false);
-        LOG.info("ApportionFeature Flag Value in CardPaymentController : {}", apportionFeature);
+        boolean apportionFeature = featureToggler.getBooleanValue(apportionFeatureValue,false);
```

Comment:
Apportion feature flag in prod is named as 'apportion-feature'. are we changing the flag?

Reply:
It is just a variable name. Look at line no 74 value doesn't change. This is to remove the sonar lint issue.